src/main/java/com/autotune/common/experiments/TrialDetails.java

```diff
@@ -30,15 +32,19 @@ public class TrialDetails {
      */
     @SerializedName("config")
     private ContainerConfigData configData ;
-
+    private String trailID;
```

Comment:
I am assuming this is the previous `trialID` and not `trailID`. In any case, let us remove this field altogether as strictly from an EM Point of View, `experiment_name` along with `trial_number` should be able to uniquely identify a trail.

Reply:
Thanks Dino . Yes we  no more using TrialID  ..It is suppose to be TrialNumber I just renamed it .